lib/ret_web/channels/hub_channel.ex

```diff
@@ -402,6 +484,19 @@ defmodule RetWeb.HubChannel do
 
   def handle_out("mute", _payload, socket), do: {:noreply, socket}
 
+  defp should_broadcast(%{"networkId" => network_id, "template" => naf_template}, socket) do
+    account = Guardian.Phoenix.Socket.current_resource(socket)
+    hub = socket |> hub_for_socket
+    is_creator = socket.assigns.created_objects |> Enum.member?(network_id)
+
+    cond do
+      naf_template |> String.ends_with?("-media") -> is_creator or account |> can?(spawn_and_move_media(hub))
+      naf_template |> String.ends_with?("-camera") -> account |> can?(spawn_camera(hub))
+      naf_template |> String.ends_with?("-pen") -> account |> can?(spawn_drawing(hub))
```

Comment:
this means you can't use another users pen or camera if you don't have spawn permissions either right? that seems OK but we'll want to make the client updated.